js/main.js

```diff
@@ -23,6 +30,165 @@ function addCats(eachCat) {
+  infoValues.appendChild(weightCol);
+
+  var lifeSpanCol = document.createElement('div');
+  lifeSpanCol.className = 'col-half';
+
+  var lifeSpanName = document.createElement('h2');
+  lifeSpanName.textContent = 'Life Span';
+
+  var lifeSpanValue = document.createElement('p');
+  lifeSpanValue.textContent = eachCat.life_span + ' years';
+  lifeSpanValue.className = 'info-value';
+
+  lifeSpanCol.appendChild(lifeSpanName);
+  lifeSpanCol.appendChild(lifeSpanValue);
+  infoValues.appendChild(lifeSpanCol);
+
+  var energyLevelCol = document.createElement('div');
+  energyLevelCol.className = 'col-half';
+
+  var energyLevelName = document.createElement('h2');
+  energyLevelName.textContent = 'Energy Level';
+
+  var energyLevelValue = document.createElement('div');
+  energyLevelValue.className = 'paws';
+  for (var i = eachCat.energy_level; i > 0; i--) {
+    var paw = document.createElement('i');
+    paw.className = 'fas fa-paw';
+    energyLevelValue.appendChild(paw);
+  }
+
+  energyLevelCol.appendChild(energyLevelName);
+  energyLevelCol.appendChild(energyLevelValue);
+  infoValues.appendChild(energyLevelCol);
+
+  var affectionCol = document.createElement('div');
+  affectionCol.className = 'col-half';
+
+  var affectionName = document.createElement('h2');
+  affectionName.textContent = 'Affection';
+
+  var affectionValue = document.createElement('div');
+  affectionValue.className = 'paws';
+  for (i = eachCat.affection_level; i > 0; i--) {
+    paw = document.createElement('i');
+    paw.className = 'fas fa-paw';
+    affectionValue.appendChild(paw);
+  }
+
+  affectionCol.appendChild(affectionName);
+  affectionCol.appendChild(affectionValue);
+  infoValues.appendChild(affectionCol);
+
+  var childFriendlyCol = document.createElement('div');
+  childFriendlyCol.className = 'col-full';
+
+  var childFriendlyName = document.createElement('h2');
+  childFriendlyName.textContent = 'Child Friendly';
+
+  var childFriendlyValue = document.createElement('div');
+  childFriendlyValue.className = 'paws';
+  for (i = eachCat.child_friendly; i > 0; i--) {
+    paw = document.createElement('i');
+    paw.className = 'fas fa-paw';
+    childFriendlyValue.appendChild(paw);
+  }
+
+  childFriendlyCol.appendChild(childFriendlyName);
+  childFriendlyCol.appendChild(childFriendlyValue);
+  infoValues.appendChild(childFriendlyCol);
+  infoRow.appendChild(infoValues);
+  infoViewContainer.appendChild(infoRow);
+
+  return infoViewContainer;
+}
+
+function getInfo(event) {
+  var infoViewsList = document.querySelectorAll('.info-view-container');
+  if (event.target.className === 'cat-name' || event.target.className === 'cat-img') {
+    for (var i = 0; i < infoViewsList.length; i++) {
+      if (infoViewsList[i].dataset.view === event.target.parentNode.dataset.view) {
+        breedsView.className = 'breeds-view hidden';
+        infoViews.className = 'info-views';
+        infoViewsList[i].className = 'info-view-container';
```

Comment:
Rather than having one of these generated for each result ahead of time and just hiding and showing it, you should have one element that will hold the detail content, and just hide and show that element as needed, and change the value of the elements within it.